admin_manual/installation/linux_installation.rst

```diff
@@ -2,46 +2,59 @@
 Preferred Linux Installation Method
 ===================================
 
+We recommend the installation from the tar ball for large productive environments 
+and experienced Linux administators. This is especially useful in installations, where 
+webserver, storage and database are on separate machines.
+In this scenario, all dependencies and requirements are managed by the package management 
+of your OS, while the ownCloud code itself is maintained in a sequence of simple steps 
+as documented in our instructions for the `Manual ownCloud Upgrade <https://doc.owncloud.com/server/9.0/admin_manual/maintenance/manual_upgrade.html>`_.
+
+The package installation is for smaller setups, where ease of use matters more than 
+stability of operation.
+
 Changes in 9.0
 --------------
 
 Linux distribution packages (from `Open Build Service`_) have been divided into 
 multiple packages for ownCloud 9: ``owncloud-deps`` and ``owncloud-files``. 
 
-The ``owncloud-files`` package installs only ownCloud, with no Apache, database, 
+The ``owncloud-files`` package installs only ownCloud, without Apache, database, 
 or PHP dependencies. 
 
 The ``owncloud-deps`` packages install all dependencies: Apache, PHP, and MySQL. 
 ``owncloud-deps`` is not intended to be installed by itself, but rather is 
 pulled in by the metapackage ``owncloud``. 
 
-Install ``owncloud`` to get a complete installation with dependencies.
-
-Split packages are available for the following Linux distributions:
-
-* CentOS 7                      
-* Debian 8 
-* RHEL 7 
-* SLES 12 
-* Ubuntu 14.04, 15.10
-* openSUSE 13.2, Leap 42.1
+Install ``owncloud`` to get a complete installation with all dependencies.
 
 ``owncloud-files`` is available for the following distributions, but not 
-``owncloud-deps``. You will have to install your own LAMP stack first. This 
+``owncloud-deps``.
+You will have to install your own LAMP stack first. This 
 allows you to create your own custom LAMP stack without dependency conflicts 
 with the ownCloud package. Browse 
 `<http://download.owncloud.org/download/repositories/9.0/owncloud/>`_ to find 
 the ``owncloud-files`` package for your distro:
 
-* CentOS 6
-* Debian 7
-* RHEL 6
-* Ubuntu 12.04, 14.10
+* Ubuntu 14.04, 16.04
+* Debian 7, 8
+* RHEL 6, 7
+* CentOS 6 SCL, 7
+* SLES 12, 12 SP1
+* openSUSE 13.2, Leap 42.1
+
+owncloud packages with dependencies are available for the following Linux distributions:
+
+* Ubuntu 14.04, 16.04
+* Debian 8
+* RHEL 7
+* CentOS 7
+* SLES 12
+* openSUSE 13.2, Leap 42.1
+
 
```

Comment:
Should not be:  CentOS 7, CentOS_6_SCL_PHP54, Debian 8, RHEL 7, SLES 12, 12 SP1, Ubuntu 14.04, 15.10, openSUSE 13.1, 13.2, Leap 42.1 ? This is what I see in our Package Installation README.